director/accounts/templates/accounts/account_profile.html

```diff
@@ -20,7 +20,7 @@
                     <div class="level-right">
                         <p class="level-item">
                             <a class="button is-primary"
-                               href="{% url 'project_create' %}?account_id={{ account.pk }}">
+                               href="{% url 'project_create' %}?account_id={{ account.name }}">
```

Comment:
This needs to stay as `account.pk` as it's used for a form value.

Reply:
:scream_cat: Well spotted; fixed.